steps/movewindowsdvdisotobuilddirectory/script.txt

```diff
@@ -0,0 +1,20 @@
+$CURRENT_PATH = "C:\software\build-{newOsNode.fqn}"
+
+Set-Location "${CURRENT_PATH}"
+
+$ISO = (dir *.ISO).name
+
+$DEST_FOLDER = "{automationWorkerWindowsBaseDirectory}\build-{newOsNode.fqn}"

```

Comment:
@h4xhor, are you removing the automationWorkerWindowsBaseDirectory parameter?

Reply:
Hi @brentonford , no. I am still using it.

Reply:
Hi @brentonford , I've removed parameter `automationWorkerWindowsBaseDirectory` and hard coded it as `C:/attuneautomationworker`.